app/views/trainings/result.html.erb

```diff
@@ -0,0 +1,18 @@
+<table class="table table-hover">
+  <thead>
+    <th> Name </th>
+    <th> Date </th>
+    <th> Elapsed Time </th>
+    <th> Discipline </th>
+  </thead>
+  <tbody>
+    <% training.each do |training| %>
+      <tr>
+        <td><%= link_to training.name, user_training_path(current_user, training) %></td>
+        <td><%= training.dateoftraining %></td>
+        <td><%= training.trainingtime_minutes %></td>
```

Comment:
this attribute is associated to `training` anyway, so it should be something like `time_in_minutes`